corehq/apps/domain/templates/domain/internal_calculations.html

```diff
@@ -39,7 +39,7 @@
     </script>
 {% endblock %}
 
-{% block main_column %}
+{% block page_content %}
     <div class="form form-horizontal">
```

Comment:
the form controls will also have to be modified: http://getbootstrap.com/css/#forms